<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWin</class>
 <widget class="QDialog" name="PreferencesWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_User">
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Default nick:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leNick"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Alternative nick:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Default ident:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leNickFix"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leIdent"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Quit:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Real name:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Ignored_text">
      <attribute name="title">
       <string>Ignored text</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Highlights">
      <attribute name="title">
       <string>Highlights</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_IRC">
      <attribute name="title">
       <string>IRC</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Ignored IRC numerics:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="text">
            <string>110</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Establish-connection timeout</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Connection timeout</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>client -&gt; server ping delay</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Use lazy ping</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Encoding</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_Enc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_24">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Changing encoding will not affect existing connections, it only applies for local connections, not connections over bouncer. Default encoding is depending on Qt implementation, but it's almost always UTF-8.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Colors">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Skin</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_SkinName"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_SkinFont"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_SkinSz"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_IC">
           <property name="text">
            <string>System info</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pushButton_CA">
           <property name="text">
            <string>+a</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="pushButton_TC">
           <property name="text">
            <string>Timestamp</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_UC">
           <property name="text">
            <string>Unread</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QPushButton" name="pushButton_C5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="pushButton_C3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="pushButton_C8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Text color</string>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QPushButton" name="pushButton_C15">
           <property name="text">
            <string>15</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QPushButton" name="pushButton_C14">
           <property name="text">
            <string>14</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="pushButton_AC">
           <property name="text">
            <string>Away</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_SC">
           <property name="text">
            <string>System color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_EC">
           <property name="text">
            <string>Error</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="pushButton_C11">
           <property name="text">
            <string>11</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="pushButton_C1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="pushButton_CV">
           <property name="text">
            <string>+v</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_CQ">
           <property name="text">
            <string>+q</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="pushButton_C12">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="pushButton_CH">
           <property name="text">
            <string>+h</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="pushButton_C7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Background color</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="pushButton_C6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QPushButton" name="pushButton_C10">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="pushButton_C2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QPushButton" name="pushButton_C9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QPushButton" name="pushButton_C13">
           <property name="text">
            <string>13</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_HC">
           <property name="text">
            <string>Highlight</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="pushButton_CO">
           <property name="text">
            <string>+o</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_WC">
           <property name="text">
            <string>Warning</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="pushButton_C4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_UC_2">
           <property name="text">
            <string>User (in text)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_LC">
           <property name="text">
            <string>Link</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Transparency</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note:&lt;/span&gt; you must restart grumpy if you modify the skin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Duplicate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_System">
      <attribute name="title">
       <string>System</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Disable all checks for valid SSL certificate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSplitMs">
         <property name="text">
          <string>Split message that are longer than:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="text">
          <string>300</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Execute these commands upon start:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditAutorun"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Standard header</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_StandardH"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_LabelH">
           <property name="text">
            <string>Labeled header</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Channel header</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_LabeledH"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_ChannelH"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Scrollback">
      <attribute name="title">
       <string>Scrollback</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Text format:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_FormatText"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Message:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_FormatMsg"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Action:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_FormatActn"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_FormatNotc">
           <property name="text">
            <string>Notice:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_FormatNt"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_22">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Link separators</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_23">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Generic separators</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_LS">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_Sep">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesWin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
